components/common/Modal/index.tsx

```diff
@@ -35,7 +35,7 @@ const ModalOverlay = styled.div`
 `
 
 const ModalContent = styled.div`
-  width: 85%;
-  max-width: 550px;
+  width: 80%;
```

Comment:
```suggestion
  width: calc(100% - 24px);
```
이렇게 규격을 잡는건 어떨까요?
지금은 모바일 규격에서 봤을 때 피그마의 디자인과 상이하게 느껴지네요! (여백이 너무 커서)
<img width="343" alt="스크린샷 2023-08-09 오후 11 10 08" src="https://github.com/miind-palace/mind-palace-FE/assets/90050960/cf3f2678-203e-4516-95f5-5b56827c466b">

but.. 어차피 허니비의 UI/UX 개선 버전에 맞추어 수정 작업을 한 번 싹 진행해야 할 것 같아서, 이건 지금은 일단 미뤄두고 UI 수정 작업 진행할 때 같이 고쳐도 될 것 같습니다~
이 코멘트의 적용 여부는 담의 선택에 맡길게요